Formula/a/afl++.rb

```diff
@@ -0,0 +1,48 @@
+class Aflxx < Formula
+  desc "American Fuzzy Lop++"
+  homepage "https://aflplus.plus"
+  url "https://github.com/AFLplusplus/AFLplusplus/archive/refs/tags/v4.21c.tar.gz"
+  sha256 "11f7c77d37cff6e7f65ac7cc55bab7901e0c6208e845a38764394d04ed567b30"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+  depends_on "coreutils" => :build
+  depends_on "make" => :build
```

Comment:
Can't we rely on system `make`?

Reply:
Humm I grabbed the dependencies from their build instructions. Seems the build works without them so they are removed.